test/unit/job_invocation_composer_test.rb

```diff
@@ -21,16 +21,16 @@
     User.current.save
   end
 
-  let(:testing_job_template_1) { FactoryGirl.create(:job_template, :job_name => 'testing_job_template_1', :name => 'testing1', :provider_type => 'Ssh') }
```

Comment:
Eh, this one is kind of dumb but Minitest 5 doesn't allow to write `let(:test**` and fails with ``let': name cannot begin with "test" (ArgumentError)`